Classes/Csv/AbstractRegistrationListView.php

```diff
@@ -101,7 +103,15 @@ public function render()
 
         $allLines = array_merge([$this->createCsvHeading()], $this->createCsvBodyLines());
 
-        return $this->createCsvSeparatorLine() . implode(self::LINE_SEPARATOR, $allLines) . self::LINE_SEPARATOR;
+        $allLines = $this->createCsvSeparatorLine()
+            . implode(self::LINE_SEPARATOR, $allLines)
+            . self::LINE_SEPARATOR;
+
+        /** @var HookProvider $csvHookProvider */
+        $csvHookProvider = GeneralUtility::makeInstance(HookProvider::class, RegistrationListCsv::class);
+        $allLines = $csvHookProvider->executeHookReturningModifiedValue('modifyCsv', $allLines, $this);
```

Comment:
`$allLines` now has two different meanings (all lines before and all lines after processing the hook(s)). Please either let's have two different varaiables, or directly return the hook results.